app/assets/stylesheets/govuk_publishing_components/components/_cards.scss

```diff
@@ -131,21 +131,42 @@
   }
 }
 
+// stylelint-disable declaration-no-important
 @include govuk-media-query($media-type: print) {
   .gem-c-cards__list {
     display: block;
   }
 
   .gem-c-cards__list-item {
-    padding-bottom: 0;
+    padding: 0;
   }
 
-  .gem-c-cards__sub-heading {
-    margin-top: govuk-spacing(4);
-    margin-bottom: govuk-spacing(1);
+  .gem-c-cards__list-item-wrapper {
+    padding: 5mm 0;
+  }
+
+  .gem-c-cards__description {
+    margin: 0;
   }
 
-  .gem-c-cards__link::before {
-    display: none;
+  .gem-c-cards__link {
+    &,
+    &:link,
+    &:link:visited {
+      color: $govuk-print-text-colour !important;
+    }
+
+    &::before {
+      display: none;
+    }
+
+    &::after {
```

Comment:
What do these styles do? It doesn't look like the after element contains anything, looks like it's just there as an overlay on the link so that the clickable area is expanded. I think you can set this to just `display: none` and it'll work fine?

Reply:
In the print view, the `::after` element changes from the overlay to the printable URL link. These styles are making the printable link a bit smaller and on their own line, to be consistent with the same treatment in other components.